vorbis-2017-12-11/build.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# Copyright 2017 Google Inc. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+. $(dirname $0)/../custom-build.sh $1 $2
+. $(dirname $0)/../common.sh
+
+readonly INSTALL_DIR="$PWD/INSTALL"
+
+build_ogg() {
+  rm -rf BUILD/ogg
+  mkdir -p BUILD/ogg $INSTALL_DIR
+  cp -r SRC/ogg/* BUILD/ogg/
+  (cd BUILD/ogg && ./autogen.sh && ./configure \
+    --prefix="$INSTALL_DIR" \
+    --enable-static \
+    --disable-shared \
+    --disable-crc \
+    && make clean && make -j $JOBS && make install)
+}
+
+build_vorbis() {
+  rm -rf BUILD/vorbis
+  mkdir -p BUILD/vorbis $INSTALL_DIR
+  cp -r SRC/vorbis/* BUILD/vorbis/
+  (cd BUILD/vorbis && ./autogen.sh && ./configure \
+    --prefix="$INSTALL_DIR" \
+    --enable-static \
+    --disable-shared \
+    && make clean && make -j $JOBS && make install)
+}
+
+download_fuzz_target() {
+  [[ ! -e SRC/oss-fuzz ]] && \
+    git clone -n https://github.com/google/oss-fuzz.git SRC/oss-fuzz
```

Comment:
why not get_git_revision?

Reply:
`get_git_revision` downloads all files in the repo.  I only wanted to download one specific file.

I could use get_git_revision for simplicity, at the cost of slightly more space used.

Reply:
ok